src/main.ts

```diff
@@ -15,11 +15,11 @@ async function run() {
 }
 
 function checkEnvironment() {
-  const supportedPlatforms : string[] = ["linux/x64"];
-  const platform : string = `${process.platform}/${process.arch}`;
+  const supportedPlatforms: string[] = ['linux/x64'];
+  const platform = `${process.platform}/${process.arch}`;
```

Comment:
Why remove the type annotation?

Reply:
It's the application of [@typescript-eslint/no-inferrable-types](https://github.com/typescript-eslint/typescript-)eslint/blob/main/packages/eslint-plugin/docs/rules/no-inferrable-types.md

Reply:
Ah, that makes sense.